src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+                System.out.println("\t event <task name> /at <date/time>");
+            } catch (IOException e) {
+                System.out.println("\t ☹ OOPS!!! File is not found!");
+            } catch (EmptyListException e) {
+                System.out.println("\t ☹ OOPS!!! Your task list is empty! Try adding a task first!");
+            } catch (NumberFormatException e) {
+                System.out.println("\t ☹ OOPS!!! You entered an invalid number!");
+            }
+            input = in.nextLine(); // Get string input
+        }
+        ui.printExitMessage(); // Exit
+    }
+
+    public static void main(String[] args) throws IOException {
+        new Duke("data/duke.txt").run();
+    }
+
+    private void manageCommand(String input, String[] words) throws NumberFormatException, EmptyTaskException, EmptyListException, InvalidTaskIndexException, InvalidCommandException, InvalidDeadlineException, InvalidEventException {
+        ArrayList<Task> taskList = tasks.getTaskList();
+        if (words[0].equals("help")) {
+            ui.printHelpMessage();
+        } else if (words[0].equals("list")) { // List tasks
+            tasks.listCommand();
+            ui.printListMessage(taskList);
+        } else if (words[0].equals("done")) { // Mark task as done
+            parser.parseDone(words);
+            int taskListSize = tasks.getTaskListSize();
+            int taskIndex = parser.parseTaskIndex(words, taskListSize);
+            tasks.markDone(taskIndex);
+            ui.printDoneMessage(taskList, taskIndex);
+        } else if (words[0].equals("deadline")) { // Deadline
+            String[] deadlineWords = parser.parseDeadline(words);
+            tasks.addDeadline(deadlineWords);
+            ui.printBorder();
+            ui.printTaskAdded(taskList);
+            ui.printListCount(taskList);
+            ui.printBorder();
+        } else if (words[0].equals("event")) { // Event
+            String[] eventWords = parser.parseEvent(words);
+            tasks.addEvent(eventWords);
+            ui.printBorder();
+            ui.printTaskAdded(taskList);
+            ui.printListCount(taskList);
+            ui.printBorder();
+        } else if (words[0].equals("todo")) { // ToDo
+            String toDoTask = parser.parseToDo(words);
+            tasks.addToDo(toDoTask);
+            ui.printBorder();
+            ui.printTaskAdded(taskList);
+            ui.printListCount(taskList);
+            ui.printBorder();
+        } else if (words[0].equals("delete")) {
+            parser.parseDelete(words);
+            int taskListSize = tasks.getTaskListSize();
+            int taskIndex = parser.parseTaskIndex(words, taskListSize);
+            ui.printBorder();
+            ui.printDeleteMessage(taskList, taskIndex);
+            tasks.deleteTask(taskIndex);
+            ui.printListCount(taskList);
+            ui.printBorder();
+        } else {
+            throw new InvalidCommandException();
+        }
```

Comment:
This is a good implementation of control flow. However, since there are multiple conditions that you are looking at, you may want to use a switch statement as it is more succinct.